src/pudl/output/eia860.py

```diff
@@ -368,13 +367,15 @@ def generators_eia860(
 
 
 def fill_generator_technology_description(gens_df: pd.DataFrame) -> pd.DataFrame:
-    """Fill in missing ``technology_description`` based on generator and energy source.
+    """Fill in missing ``technology_description`` based by backfilling & unquie mapping.
```

Comment:
Tiny comment, but unique is spelled wrong

Reply:
lol of course ty!